frontend/main-app/src/components/Login/Login.test.tsx

```diff
@@ -5,7 +5,7 @@ import { Login } from "./Login";
 import { MemoryRouter } from "react-router";
 
 describe("Login component", () => {
-  it("loads and displays Login component", () => {
+  it("loads and displays Login component", async () => {
```

Comment:
I don't think async is required when using getByRole and getByLabelText
https://testing-library.com/docs/dom-testing-library/api-async/